bitcoin/script/miniscript.h

```diff
@@ -293,6 +294,41 @@ struct StackSize {
     StackSize(MaxInt<uint32_t> in_sat, MaxInt<uint32_t> in_dsat) : sat(in_sat), dsat(in_dsat) {};
 };
 
+struct TimeLockInfo {
+    //! csv with heights
+    bool csv_h;
+    //! csv with times
+    bool csv_t;
```

Comment:
Oops, I will change this later for sure. Looks like github UI does `_t` in variable name for a good reason.

Reply:
Ending a name with `_t` is actually UB, I believe.